lib/generator._js

```diff
@@ -461,6 +454,9 @@ ScriptGenerator.prototype.generateAspNetCoreScript = function (templateFileName,
   argNotNull(templateFileName, 'templateFileName');
 
   var lowerCaseScriptType = this.scriptType.toLowerCase();
+  var fixedRestore = this.scriptType === ScriptType.batch ? fixPathSeperatorToWindows(options.restore) : fixPathSeperatorToUnix(options.restore);
```

Comment:
I just found that PowerShell on Windows correctly handles forward slashes as the path separator. So we don't have any problem at all.